bin/rpm.go

```diff
@@ -355,10 +355,14 @@ fi
 
 	r.AddPostun(`
 if [ -f /bin/systemctl ] ; then
+    if [ $1 = 0 ] ; then
 	rm /etc/systemd/system/velociraptor_client.service
+    fi
 else
+    if [ $1 = 0 ] ; then
 	/sbin/service velociraptor start  > /dev/null 2>&1 || :
```

Comment:
I think this needs to be split into upgrade and uninstall parts as well - no point to try to start the service when it is uninstalled - we should remove on uninstall and start on upgrade.

Or maybe move this `/sbin/service velociraptor start  > /dev/null 2>&1 || :` to the preun above?